data/json/items/armor/ballistic_armor.json

```
@@ -176,7 +176,7 @@
     "//2": "link like: https://www.shindn.com/products/nij-iiia?variant=35614363582627&currency=USD&utm_medium=product_sync&utm_source=google&utm_content=sag_organic&utm_campaign=sag_organic&utm_campaign=gs-2019-08-03&utm_source=google&utm_medium=smart_campaign&gclid=Cj0KCQjw5-WRBhCKARIsAAId9FlGQAManRGZ0qpV9AIkD23FZfpm4aPDI4PHE9qPktZ2bdQY2NmcIVkaAs2pEALw_wcB",
     "description": "A heavy and concave ballistic plate made of tempered steel, designed to stop small-caliber rounds.  \"NIJ III PROTECTION\" is etched into the back of the plate.  This is intended to be inserted into a ballistic vest, and can't be worn on its own.",
     "weight": "2500 g",
-    "//3": "30cm x 25cm x .55cm",
+    "//3": "30cm x 25cm x .5cm",
```

I'm not sure why this was changed?

---

huh. must have been a mistake getting confused between my copy pastes. need to undo that line

---

then you can merge it here or do it when you are doing your formatting
```suggestion
    "//3": "30cm x 25cm x .55cm",
```

---

okay, i've linted the files and removed that error